internal/pkg/policy/policy_output.go

```diff
@@ -411,6 +444,28 @@ func generateOutputAPIKey(
 	)
 }
 
+func generateRemoteOutputAPIKey(ctx context.Context,
```

Comment:
Any suggestion where to put this to reuse the es client? Maybe a new service? The existing api key generation is in bulker, but I can't use that as it uses the local ES client.

Reply:
We may want to introduce a bulker manager (or map of bulkers) that can have "named" clients.
Right now it will only be used to support remote outputs in policies, but it may be useful in the future (if we ever want to try to tackle multi tenancy).

I think this item may be better to leave as tech-debt at it would be a pretty big change if we wanted to replace all our bulkers

Reply:
Yeah as far as I understood, there is one bulker now that is initialized with the local ES client, so I can't reuse it for remote ES. 
I could create another bulker instance for the remote ES client for now.